snuba/admin/views.py

```diff
@@ -87,18 +90,17 @@ def health() -> Response:
 
 @application.route("/migrations/groups")
 def migrations_groups() -> Response:
+    group_policies = get_migration_group_polices()
+    allowed_groups = group_policies.keys()
+
     res: List[Mapping[str, str | Sequence[Mapping[str, str | bool]]]] = []
-    allowed_groups: Sequence[MigrationGroup] = [
-        group
-        for group in get_active_migration_groups()
```

Comment:
actually we are going to need the `get_active_migration_groups()` because that's what the runner uses so need to add this back in